microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -200,7 +200,7 @@ If you set `AreBrowserAcceleratorKeysEnabled` to `FALSE`, the following addition
 | Find Previous | **Shift+F3** |
 | Back | **Alt+Left Arrow**, **Browser Back Button Key** |
 | Forward | **Alt+Right Arrow**, **Browser Forward Key** |
-| Print | **Ctrl+P** |
+| Caret Browsing | **F7** |
 | Open / Close DevTools | **Ctrl+Shift+I** |
```

Comment:
Edge > **Settings** > **Learn more** = https://go.microsoft.com/fwlink/?linkid=2165400 which incorrectly goes to
https://support.microsoft.com/en-us/microsoft-edge/accessibility-features-in-microsoft-edge-4c696192-338e-9465-b2cd-bd9b698ad19a#caretwindows
which section isn't found, so defaults to top of page instead of scrolling down to section.
fwklink should be changed from #caretwindows to #caret_browsing

Reply:
email sent, suggesting entering a work item.
The present comment thread can be Resolved.